config/ui-extensions/apirules/details

```diff
@@ -11,8 +11,8 @@ header:
     source: 'status.APIRuleStatus.code ? status.APIRuleStatus.code : "UNKNOWN"'
     description: status.APIRuleStatus.desc
   - name: host
-    source: spec
-    widget: APIRuleHost
+    widget: ExternalLink
+    source: '$virtualServices().items[0].spec.hosts[0]'
```

Comment:
This will only work when the VS is updated with the up to date host.
If, for example an update would cause the APIRule to go to `error` and not update the VS, it will be showing the host from the VS, not the actual one on APIRule

Simple scenario:
- create APIRule 1 with host a.example.com
- create APIRule 2 with host b.example.com
- update APIRule 2 with host a.example.com

The APIRule 2 will go into error, and the shown host will be `b.example.com`

Reply:
While this is true, I think the only sane solution to have this link is to derive it from the VS, because deriving it from the APIRule and Gateway resources is error-prone and very complex. Especially since you can define the gateway in different format in the APIRule.

Also the whole UI must be reimagined for APIRule v1beta2, since we support multiple hosts.

Another solution could be to have the host field render the actual value and only if APIRule is in Ready state replace it with a link taken from the VS.
What do you think? @barchw

Reply:
Since all our UI tests are e2e right now and I think it's okay. I don't think we'll change this in the next months, especially since to really decouple them a big change in how Busola processes UI supplied by modules would be required.